ACE/ace/Module.h

```diff
@@ -59,6 +59,8 @@ class ACE_Export ACE_Module_Base
     /// Indicates that close() should not delete any Tasks.
     M_DELETE_NONE = 4
   };
+
+  inline virtual ~ACE_Module_Base (void) {}
```

Comment:
Make use of C++11 default, remove redundant void

Reply:
Done